modules/nw-metalLB-upgrading-operator.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+
+[id="upgrading-metalLB-configured-for-a-specific-namespace_{context}"]
+= Upgrading MetalLB when previously installed to a specific namespace
+
+Follow this procedure to if you are upgrading from a prior version and the Operator has been installed to a specific namespace.
+
+.Prerequisites
+
+* Install the {product-title} CLI (oc).
+* Log in to the OpenShift cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Determine the unique name associated with the OperatorGroup:
++
+[source,terminal]
+----
+$ oc describe -n metallb-system og metallb-system
```

Comment:
This goes under the assumption that the user deployed the operator in the metallb-system namespace, which could not be the case. Also, I'd be explicit about the fact that the new installation must be deployed in the same namespace.